src/client/components/SubmitFormSingleInputComponent/SubmitFormSingleInput.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ButtonComponent from '../SubmitButton/SubmitButton';
-import './SingleInputFormComponent.style.css';
+import './SubmitFormSingleInput.style.css';
+import Popup from '../Modal/ContentModal.js';
+
+const toggleModal = () => {
+  document.querySelector('.modalbox').classList.toggle('showmodal');
+};
```

Comment:
Not the correct "react way" to do things as we discussed before. Please add a todo.

Reply:
As discussed we are not using modal for validation so it is removed, but i have modified modal calling technique according to react using state.